templates/settingsForm.tpl

```diff
@@ -8,53 +8,41 @@
  * Registration Notification plugin settings
  *
  *}
-<div id="registrationNotificationSettings">
-<div id="description">{translate key="plugins.generic.registrationNotification.description"}</div>
-
-<h3>{translate key="navigation.settings"}</h3>
-
+<script src="{$pluginJavaScriptURL}/RegistrationNotificationFormHandler.js"></script>
 <script>
 	$(function() {ldelim}
 		// Attach the form handler.
-		$('#registrationNotificationSettingsForm')
-			.on('click', '.remove-button', function() {ldelim}
-				$(this).parents('.section').remove();
-			{rdelim})
-			.on('click', '.insert-button', function() {ldelim}
-				var section = $(this).parents('.section');
-				var clone = section.clone();
-				section.find(':input:not(button)').val('');
-				clone.find('.insert-button')
-					.removeClass('insert-button pkp_button_primary')
-					.addClass('remove-button')
-					.text('{translate key="common.remove"}');
-				clone.insertBefore(section);
-			{rdelim})
-			.pkpHandler('$.pkp.controllers.form.AjaxFormHandler');
+		$('#registrationNotificationSettingsForm').pkpHandler(
+			'$.pkp.controllers.form.registrationNotification.RegistrationNotificationFormHandler', 
+			{ldelim}removeCaption: '{translate key="common.remove"}'{rdelim}
```

Comment:
This will cause JS errors if the locale key contains an apostrophe. Suggest using e.g.:
```
{ldelim}removeCaption: {translate|json_encode key="common.remove"}{rdelim}
```
(Note the removal of single quotes -- `json_encode` will take care of this.)